e2e/tests/onboarding.e2e.ts

```diff
@@ -1,6 +1,6 @@
 import {device} from 'detox';
 import setLocation from '../utils';
-import {tapById} from '../utils/interactionHelpers';
+import { tapById, tapByText } from "../utils/interactionHelpers";
```

Comment:
Linting error?

```suggestion
import {tapById, tapByText} from '../utils/interactionHelpers';
```

But are you even using `tapByText` here? If not:

```suggestion
import {tapById} from '../utils/interactionHelpers';
```

Reply:
Takk! Fixet opp i non flere slike issuer i ny commit